s3/pkg/meta/types/object.go

```diff
@@ -186,7 +184,7 @@ func (o *Object) encryptSseKey() (err error) {
 	return nil
 }
 
-func (o *Object) GetVersionId() string {
+/*func (o *Object) GetVersionId() string {
```

Comment:
Can we remove this commented code? Anyways, this will be coming in another PR, right?